kolibri/core/assets/src/composables/useMinimumKolibriVersion.js

```diff
@@ -0,0 +1,35 @@
+import { computed } from 'kolibri.lib.vueCompositionApi';
+
+export default function useMinimumKolibriVersion() {
+  const isMinimumKolibriVersion = computed(() => {
+    return function(version, majorVersion = 0, minorVersion = 15, revisionVersion = null) {
+      /*
+      defaults to kolibri 0.15.x
+      If values are not provided for revisionVersion, then any values are allowed.
+      This means that if no value is provided for revisionVersion then
+      alpha and beta versions will be permitted.
+      Just providing 0 as majorVersion and 15 as minorVersion would allow any version
+      greater than or equal to 0.15 (including any alphas or betas)
+      */
+      const v = version.split('.');
```

Comment:
I was talking about t his with Richard, if we want to have a complete flexibility on the version naming,  we should use something like https://www.npmjs.com/package/compare-versions 
However, in this case, as we're who are the users of the function and kolibri is going to be a '0.0.0' version always we don't need to add one more dependency. So, yes , you are right for that case... but that case is not going to happen.

Reply:
No need for complete flexibility. That's up to you. I imagine that at least communicating that such a decision has been made would help, with a comment or the parameter check. v1 was just one example, this will currently silently fail for non-v1 values too, like  `isMinimumKolibriVersion.value('0.16', 0, 15, 0)` returns `false` whereas `isMinimumKolibriVersion.value('0.16.0', 0, 15, 0)` returns `true`.

Reply:
the thing is that this is going to be used to check the version provided by kolibri, and kolibri will always provide three digits. I can add a comment in the function but, while this is been done only to be used inside kolibri, there should not be possible error.

Reply:
Everything's possible, Jose, Kolibri is written by humans, not machines (at least not yet) ;-)) Leaving philosophy behind, this is not a blocker when compared to console errors, so it's up to you.

Reply:
yes, you're right. I try to keep it simple (or using the external library would be a better option) but throwing an error as you suggest is harmless if we keep the format as it's supposed and does not add complexity. Done!